apcd_cms/src/apps/utils/apcd_database.py

```diff
@@ -152,6 +152,38 @@ def get_user_role(user):
         if conn is not None:
             conn.close()
 
+def get_submitter_users():
+    cur = None
+    conn = None
+    try:
+        conn = db_connect()
+        query = """
+        SELECT DISTINCT submitter_users.submitter_id,
+        submitter_users.user_id,
+        submitter_users.user_number,
+        users.user_email,
+        users.user_name,
+        submissions.payor_code
+        FROM submitter_users
+        JOIN users
+           ON submitter_users.user_id = users.user_id
+           AND submitter_users.user_number = users.user_number
+        JOIN submissions
+            ON submitter_users.submitter_id = submissions.submitter_id
+            ORDER BY users.user_id;
```

Comment:
submitter_users.user_id is in SELECT, so order by that one
```suggestion
            ORDER BY submitter_users.user_id;
```